### PR TITLE
fix: add cross compatibility to dev script of package file

### LINKED ISSUE #182: "NEXT_PUBLIC_E2E_TEST" is not recognized as internal or external command
After installing the dependencies and copying the .env files, in windows 10, with docker I get the error: "NEXT_PUBLIC_E2E_TEST" is not recognized as internal or external command. I looked for information in the configuration pages and I did not find what I should put as a value in the .env file for that variable or what dependency I would need to install

### PR DESCRIPTION
**Pull request by Jorgelig**
fixes #182

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "docker:up": "docker compose -f docker-compose.dev.yml up -d && sleep 5",
     "docker:nuke": "docker compose -f docker-compose.dev.yml down --volumes --remove-orphans",
     "lint": "turbo run lint",
-    "dev": "pnpm docker:up && NEXT_PUBLIC_E2E_TEST=false turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
+    "dev": "pnpm docker:up && cross-env NEXT_PUBLIC_E2E_TEST=false && turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
```
Comment (baptisteArno) [inline package.json:15]:
```suggestion
    "dev": "pnpm docker:up && cross-env NEXT_PUBLIC_E2E_TEST=false turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
```

It should work as expected without chaining a `&`

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
     "dev:mocking": "pnpm docker:up && NEXT_PUBLIC_E2E_TEST=true turbo run dev --filter=builder... --filter=viewer... --parallel --no-cache",
```
Comment (baptisteArno) [inline package.json:16]: Can you also add it to this script?

**Review by baptisteArno — CHANGES_REQUESTED** [review]
Thank you for this 🙏

File: package.json (hunk 1 above)
Comment (Jorgelig) [reply package.json]: done, removed `&&`

File: apps/viewer/package.json [hunk 3]
```diff
@@ -44,6 +44,7 @@
     "@types/react": "18.0.26",
     "@types/sanitize-html": "2.6.2",
     "dotenv": "16.0.3",
+    "cuid": "2.1.8",
```
Comment (baptisteArno) [inline apps/viewer/package.json:47]: Why?

File: apps/viewer/package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "license": "AGPL-3.0-or-later",
   "version": "0.1.0",
   "scripts": {
-    "dev": "ENVSH_ENV=.env.local bash ../../env.sh next dev -p 3001",
+    "dev": "cross-env ENVSH_ENV=.env.local && sh ../../env.sh next dev -p 3001",
```
Comment (baptisteArno) [inline apps/viewer/package.json:6]: Same, `&&` is not needed here :)

**Review by baptisteArno — CHANGES_REQUESTED** [review]

File: apps/viewer/package.json (hunk 3 above)
Comment (Jorgelig) [reply apps/viewer/package.json:47]: required by apps\viewer\src\features\blocks\inputs\fileUpload\fileUpload.spec.ts

**Review by baptisteArno — APPROVED** [review]

File: apps/viewer/package.json (hunk 3 above)
Comment (Jorgelig) [reply apps/viewer/package.json:47]: What do you think is needed?

**Comment by baptisteArno** [comment]
There are some things missing, can you allow me to push a new commit to your branch? Then I'll be able to merge it 👌

**Comment by Jorgelig** [comment]
> There are some things missing, can you allow me to push a new commit to your branch? Then I'll be able to merge it 👌

of course, tell me how can I help

**Pull request closed without merging**